### ISSUE TITLE
Any idea's of self-upgrading gheed's fortune?

### ISSUE THREAD

**Issue by DarkHorseDre** [kolton/d2bot-with-kolbot#2529]
Hi guys,
I don't want to reinvent the wheel so searched now asking first:

I want a single bot to check the found gheeds against the one it is using/in stash, and switch it if it is 'better'.
I see pickit.js currently does nothing with the new gheed it bot has one (via getItem).

anyone doing this/seen this? or do I need to write it? :)

**Comment by ARkaine2** [comment]
I don't have any useful sugesstions, but I thought about same thing for a long time and +1 to your ask.

Probably bot can't do anything with locked part of inventory (or maybe I'm mistaken).

Seems that bot should be able to throw out Gheed's Fortune from inventory, pick new/unid, ID it and compare them.

If old one is worse, keep new (better) one.
If old one is better, throw out new one and come back old one.

This should be done in "field ID" mode.

**Comment by DarkHorseDre** [comment]
Yeah the logic is straight forward, just trying to ensure someone else hasnt already done this and willing to share as there's no point reinventing the wheel!

I'm thinking doing it in public games (realm games with no password) is not worth the risk from the highwaymen.

Locked inventory should be ok to override - I've seen it done.

doing it in the field of course - you cant get both gheeds to town with 1 toon who already has a gheed! (I would override fieldID - it is an independent function).

**Comment by DarkHorseDre** [comment]
ok finally took another look at this - have it working in single player, need idea's of how to test in the wild.

problem is i cant easily simulate dropping a gheeds online - any ideas @5noop or anyone?
or do I just need to farm for one and hope for the best?!  :(

https://youtu.be/DCpvnA7M7Jk

**Comment by DarkHorseDre** [comment]
someone asked about this on YT for some reason.. this is still in testing - some bugs the size of ... uff
